transport_nantes/topicblog/tests_topic_blog.py

```diff
@@ -1048,3 +1048,82 @@ def test_send_email_form(self):
 
         form = TopicBlogEmailSendForm(data=form_data)
         self.assertTrue(form.is_valid())
+
+    def test_unique_slug_recipient_constraint(self):
+        url = reverse('topicblog:send_email',
+                      args=[self.email_article.slug])
+
+        form_data = {
+            "mailing_list": 'the_mailing_list_token',
+            "confirmation_box": 'on'
+        }
+        response = self.admin_client.post(url, form_data)
+        # Redirect to success url
+        self.assertEqual(response.status_code, 302)
+        # Check if the emails have been sent (one for each subscribed user
+        self.assertEqual(len(mail.outbox), 2)
+
+        # Empty outbox
+        mail.outbox = []
+
+        response = self.admin_client.post(url, form_data)
+        # Redirect to success url
+        self.assertEqual(response.status_code, 302)
+        # Check if the emails were not sent twice
```

Comment:
Check if the emails were sent once and not twice.

Lest one read that you check twice. ;-)